src/main/java/ua/comparison/image/ImageComparisonTools.java

```diff
@@ -66,9 +66,8 @@ public static void checkCorrectImageSize( BufferedImage image1, BufferedImage im
      * @return {@code true} if they' are difference, {@code false} otherwise.
      */
     public static boolean isDifferent( int x, int y, BufferedImage image1, BufferedImage image2 ){
-        boolean result = false;
-        int[] im1= image1.getRaster().getPixel( x,y,new int[3] );
-        int[] im2= image2.getRaster().getPixel( x,y,new int[3] );
+        int[] im1= image1.getRaster().getPixel( x,y,new int[4] );
```

Comment:
new int[4] instead of the new int[3] has to be because of the moving from 24-bit to 32-bit.
the same for line number 70.